wormhole-connect/src/views/v2/Bridge/AmountInput/index.tsx

```diff
@@ -136,19 +131,19 @@ const AmountInput = (props: Props) => {
         return;
       }
 
-      // Validation for the amount value
-      const amountValidation = validateAmount(
-        value,
-        tokenBalance,
-        getMaxAmt(route),
-      );
-
       dispatch(setAmount(value));
-      setValidationResult(amountValidation);
     },
     [amount, route, tokenBalance],
   );
 
+  const validationResult = useMemo(
+    () => validateAmount(amount, tokenBalance, getMaxAmt(route)),
+    [amount, tokenBalance, route],
```

Comment:
@nikarm22 how is this going to validate on wallet changes?

Reply:
'tokenBalance' is depending on token, chain and wallet, so it's enough basically.